src/Season_Counter.elm

```diff
@@ -0,0 +1,97 @@
+module Season_Counter exposing (..)
```

Comment:
The conventions dictate a name like `SeasonCounter` (no underscore).
If you must, create a `Season` package and put a `Counter` module inside it.
Also, don't expose everything.